proxylib/test/accesslog_server.go

```diff
@@ -21,12 +21,13 @@ import (
 	"path/filepath"
 	"strings"
 	"sync/atomic"
-	"syscall"
 	"time"
 
-	"github.com/cilium/proxy/go/cilium/api"
+	cilium "github.com/cilium/proxy/go/cilium/api"
```

Comment:
This also seems out-of-place if there's no corresponding line change for users of this import.

Reply:
I need to smash vscode in the head.. I am unsure why that changed... will fix.